### ISSUE TITLE
Compliant time units in CLI report and results export

### ISSUE THREAD

**Issue by kbobyrev** [sharkdp/hyperfine#71]
When passing `--export-markdown` the resulted Markdown file includes information about `Mean` and `Min..Max` measured in `ms` even when the original result report printed to stdout is presented in seconds. I think it might be better if the produced report would use the same units since it's hard to read very long `ms` numbers.

Instructions for the behavior reproduction:

```
hyperfine 'sleep 5' --export-markdown results.md
```

Command line output:

```
kbobyrev@kbobyrev ~/d/m/profile> hyperfine 'sleep 5' --export-markdown results.md
Benchmark #1: sleep 5
  Time (mean ± σ):      5.002 s ±  0.000 s    [User: 1.1 ms, System: 2.0 ms]
  Range (min … max):    5.002 s …  5.003 s
```

**results.md**:

| Command | Mean [ms] | Min…Max [ms] |
|:---|---:|---:|
| `sleep 5` | 5002.3 ± 0.2 | 5002.1…5002.7

**Comment by sharkdp** [comment]
Yes - we should definitely support this. Thank you for the feedback.

**Comment by jasonpeacock** [comment]
I'll give this a try...

It  looks like `BenchmarkResult` stores all values internally as seconds, and the other formats (`csv`, `json`) don't perform any conversion  - they always output as seconds.

This change should be just removing the `MULTIPLIER` from the `markdown` exporter, then it will have the same always-output-seconds behavior as the other format exporters. *This will change the markdown results for all users from milliseconds to seconds, a UX breaking change.*

But this doesn't fix the actual root cause of the issue - the results export may still differ in units from the CLI report because the CLI report will automatically adjust format between seconds and milliseconds if it's <1s while the results export is always seconds for all metrics.

I'll submit a feature request to add an option to choose (force) the units for both the CLI & results, then they will always match, if desired.

**Comment by sharkdp** [comment]
> It looks like BenchmarkResult stores all values internally as seconds, and the other formats (csv, json) don't perform any conversion - they always output as seconds.

CSV and JSON are formats to be automatically processed (where a hard-coded unit is the right choice, in my opinon), but Markdown is for humans.

> This change should be just removing the MULTIPLIER from the markdown exporter, then it will have the same always-output-seconds behavior as the other format exporters. This will change the markdown results for all users from milliseconds to seconds, a UX breaking change.

I would prefer if we show the same unit that is also shown in the terminal (which is automatically infered).

**Comment by jasonpeacock** [comment]
> I would prefer if we show the same unit that is also shown in the terminal (which is automatically infered).

Sure, can do. I'll update the PR with this support.

**Comment by sharkdp** [comment]
Fixed by @jasonpeacock in #84

**Comment by sharkdp** [comment]
Released in [v1.4.0](https://github.com/sharkdp/hyperfine/releases/tag/v1.4.0).
